tests/sqlparser_clickhouse.rs

```diff
@@ -211,15 +211,102 @@ fn parse_delimited_identifiers() {
 #[test]
 fn parse_create_table() {
     clickhouse().verified_stmt(r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY ("x")"#);
-    clickhouse().one_statement_parses_to(
-        r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY "x""#,
-        r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY ("x")"#,
-    );
+    clickhouse().verified_stmt(r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY "x""#);
     clickhouse().verified_stmt(
-        r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY ("x") AS SELECT * FROM "t" WHERE true"#,
+        r#"CREATE TABLE "x" ("a" "int") ENGINE=MergeTree ORDER BY "x" AS SELECT * FROM "t" WHERE true"#,
     );
 }
 
+#[test]
+fn parse_create_table_with_primary_key() {
+    match clickhouse_and_generic().one_statement_parses_to(
+        concat!(
+            r#"CREATE TABLE db.table (`i` Int, `k` Int)"#,
```

Comment:
```suggestion
            r#"CREATE TABLE db.table (`i` INT, `k` INT)"#,
```
since we're not testing the data type parsing here, we can use the same casing, and it let's us use `clickhouse_and_generic().verified_stmt()` instead?

Reply:
Makes sense.
I updated this unit-test.